mlrun/datastore/sources.py

```diff
@@ -52,6 +51,30 @@ def get_source_step(source, key_fields=None, time_field=None, context=None):
     return source.to_step(key_fields, time_field, context)
 
 
+class _MongoDBiter:
```

Comment:
either `_MongoDBIter` or my preference: `_MongoDBIterator`. depends on the naming style you see in this file...